packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -150,6 +150,9 @@ abstract class Route<T> {
   ///
   /// The returned value resolves when the push transition is complete.
   ///
+  /// [didAdd] method will be called instead of [didPush] when the route
```

Comment:
nit: comments should be grammatically correct and start with a capital letter. You could say "The [didAdd] method..."